webtop/__init__.py

```diff
@@ -9,13 +9,41 @@
 import math
 import os
 import signal
+import socket
 import time
 from threading import Event
-from typing import Dict, Collection, Optional, Deque
+from typing import Dict, Collection, Optional, Deque, List, Any, Callable
 import yaml
 from yarl import URL
 
 
+class CustomResolver(aiohttp.resolver.AbstractResolver):
+    async def close(self) -> None:
+        await self.async_resolver.close()
+
+    def __init__(self, *args, custom_mappings: Optional[Dict[str, str]] = None, **kwargs):
+        super().__init__(*args, **kwargs)  # type: ignore
```

Comment:
I am really annoyed that mypy has so much trouble with unwrapping dynamic args :(

Reply:
Yeah :/ I'm not happy with it either, but I don't know how I can get around it in a way that's not gross and hacky.